AdaptiveCards/authoring-cards/input-validation.md

```diff
@@ -29,6 +29,14 @@ An `errorMessage` property is available on all input types to specify what error
 >
 > Min and max properties (including maxLength) on some platforms may be enforced directly by the control. For example, a min property on Input.Date may be enforced by not allowing users to select a date before the minimum in a date picker. In that case, the error message may not be shown.
 
+## Labels
+
+Another property added in schema version 1.3 for all input elements is the `label` string property. Using the `label` property is the recommended way of tagging inputs in an Adaptive Card, on the one hand, it is a simple and concise way of labelling inputs for card authors, while on the other it also provides the following benefits:
```

Comment:
tagging inputs in an Adaptive Card, vis-a-vis the `placeholder` property. It is a simple and concise way of labeling inputs for card authors and has the following benefits: